File: src/targets/node/request/fixtures/query.js
const request = require('request');

const options = {
  method: 'GET',
  url: 'http://mockbin.com/har',
  qs: {foo: ['bar', 'baz'], baz: 'abc', key: 'value'}
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: src/targets/ruby/faraday/fixtures/query.rb
require 'faraday'

conn = Faraday.new(
  url: 'http://mockbin.com',
)

response = conn.get('/har') do |req|
  req.params['foo'] = ["bar","baz"]
  req.params['baz'] = 'abc'
  req.params['key'] = 'value'
end

puts response.status
puts response.body